IPython/html/static/notebook/js/main.js

```diff
@@ -76,7 +76,8 @@ require([
         events: events, 
         keyboard_manager: keyboard_manager});
     var contents = new contents.Contents($.extend({
-        events: events, config:config_section},
+          common_config: common_config
+        },
```

Comment:
Let's explicitly pass `base_url` and `common_config` rather than extending common_options here. It's the same number of lines, and explicit > implicit.